BlinkConfig/BKHosts.h

```diff
@@ -62,6 +69,9 @@ enum BKMoshPrediction {
 @property (nonatomic, strong) BKHosts *iCloudConflictCopy;
 @property (nonatomic, strong) NSString *sshConfigAttachment;
 @property (nonatomic, strong) NSString *fpDomainsJSON;
+@property (nonatomic, strong) NSString *hostAgentKeys;
```

Comment:
What is this property for?

Reply:
Definitely a left over, good you noticed.